testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -354,12 +380,20 @@ pub async fn nodes_healthcheck(nodes: Vec<&K8sNode>) -> Result<Vec<String>> {
             Box::pin(async move {
                 info!("Attempting health check: {:?}", node);
                 match node.rest_client().get_ledger_information().await {
-                    Ok(_) => {
-                        info!("Node {} healthy", node.name());
-                        Ok(())
+                    Ok(res) => {
+                        let version = res.inner().version;
+                        info!("Node {} @ version {}", node.name(), version);
+                        if version > 100 {
```

Comment:
Can you explain where is the magic number 100 comes from?

Reply:
I set it arbitrarily. The point is to ensure liveness before starting test, rather than previously where we were just checking if the REST API is available. 100 seemed like a high enough number without having to spin for too long, but it could be lower or higher tbh

Reply:
Let's add a comment here for clarity?